docs/ldml/tr35.md

```diff
@@ -2323,12 +2323,57 @@ The XML structure is stable over releases. Elements and attributes may be deprec
 
 In general, all translatable text in this format is in element contents, while attributes are reserved for types and non-translated information (such as numbers or dates). The reason that attributes are not used for translatable text is that spaces are not preserved, and we cannot predict where spaces may be significant in translated material.
 
-There are two kinds of elements in LDML: _rule_ elements and _structure_ elements. For structure elements, there are restrictions to allow for effective inheritance and processing:
+There are two kinds of elements in LDML: _rule_ elements and _structure_ elements.
 
-1.  There is no "mixed" content: if an element has textual content, then it cannot contain any elements.
+For structure elements, there are restrictions to allow for effective inheritance and processing:
+
+1.  There is no ["mixed" content](https://www.w3.org/TR/xml/#sec-mixed-content): if an element has textual content, then it cannot contain any elements.
 2.  The [[XPath](#XPath)] leading to the content is unique; no two different pieces of textual content have the same [[XPath](#XPath)].
+3.  An element that has [value attributes](#Definitions) MUST NOT also have have child elements.
+
+The following fictional (non-LDML) example illustrates the first and third restrictions on structure elements:
+
+```xml
+<parent id="abc" someValue="Some Value"> <!-- Disallowed: value attribute AND children -->
+  Here is some text   <!-- Disallowed: Mixed text and children -->
+  <child id="def"/>
+  <child id="ghi"/>
+</parent>
+```
+
+The above format repeats the "Some Value" in the xpath of each child element, making matching and inheritance processing more complex. Instead, this might be structured as:
+
+```xml
+<parent id="abc"> <!-- Allowed: 'id' is distinguishing -->
+  <someValue value="Some Value"/>
+  <text>Here is some text</text>
+  <child id="def"/>
+  <child id="ghi"/>
+</parent>
+```
+
+The following fictional (non-LDML) example illustrates the first and third restrictions on structure elements:
```

Comment:
This looks like a repeat. Also, would be better to use real elements.

Reply:
I suggest
`		<localeDisplayPattern someValueAttribute="foo">
                         some text data
			<localePattern>{0} ({1})</localePattern>
			<localeSeparator>{0}, {1}</localeSeparator>
`